### PR TITLE
EZP-29617: No route found after adding new translation

### PR DESCRIPTION
**Pull request by ViniTou**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [EZP-29617](https://jira.ez.no/browse/EZP-29617)
| **Bug/Improvement**| yes
| **New feature**    | no
| **Target version** | `7.2`
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

Needed for following PR:
https://github.com/ezsystems/ezplatform-admin-ui/pull/641

**TODO**:
- [x] Implement feature / fix a bug.
- [x] Implement tests.
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

File: eZ/Publish/Core/Repository/URLAliasService.php [hunk 1]
```diff
@@ -636,21 +641,21 @@ protected function buildSPIUrlAlias(URLAlias $urlAlias)
...
      * @param string $url
      * @param string $languageCode
+     * @param bool|null $showAllTranslations
      *
      * @return \eZ\Publish\API\Repository\Values\Content\URLAlias
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException if the path does not exist or is not valid for the given language
+     * @throws \eZ\Publish\Core\Base\Exceptions\NotFoundException
      */
-    public function lookup($url, $languageCode = null)
+    public function lookup($url, $languageCode = null, bool $showAllTranslations = null)
```
Comment (adamwojs) [inline eZ/Publish/Core/Repository/URLAliasService.php]: I think `$showAllTranslations ` parameter should be also added to the `\eZ\Publish\API\Repository\URLAliasService::lookup` implementation in the SiteAccess aware layer. 

Are you able to create a test case for proposed changes?

Comment (ViniTou) [reply eZ/Publish/Core/Repository/URLAliasService.php]: Working at it right now.

**Comment by alongosz** [comment]
I've spent some time today debugging what exactly happens here and this seems like not the solution to the actual problem we have, see [this comment](https://github.com/ezsystems/ezplatform-admin-ui/pull/641#pullrequestreview-162051272).

**Comment by alongosz** [comment]
Alternative solution not relying on these changes is available via ezsystems/ezplatform-admin-ui@1c2b094 in ezsystems/ezplatform-admin-ui#641. Closing.

**Pull request closed without merging**